.github/workflows/docker-image.yml

```
@@ -18,9 +18,19 @@ jobs:
     - name: Get current tag
       id: vars
       run: echo ::set-output name=tag::${GITHUB_REF#refs/*/}
+    - name: Extract Major and Minor Versions
+      id: version
+      run: |
+        FULL_VERSION=${{ steps.vars.outputs.tag }}
+        MAJOR_VERSION=$(echo $FULL_VERSION | cut -d '.' -f 1)
+        MINOR_VERSION=$(echo $FULL_VERSION | cut -d '.' -f 1,2)
```

I think we should prefix a `v` here. Our full version and ref name have this too, such as `v1.2.3`.

```suggestion
        MAJOR_VERSION=v$(echo $FULL_VERSION | cut -d '.' -f 1)
        MINOR_VERSION=v$(echo $FULL_VERSION | cut -d '.' -f 1,2)
```

---

Of course, this is automatically included sourced from the original tag. I accidentally tested this without a prefixed `v`. So this is all good! :smile: